matrixbot/plugins/wkbotsfeeder.py

```diff
@@ -115,47 +117,80 @@ def dispatch(self, handler=None):
             except Exception as e:
                 self.logger.error("WKBotsFeederPlugin got error in builder %s: %s" % (builder_name,e))
 
-
-    def command(self, sender, room_id, body, handler):
-        self.logger.debug("WKBotsFeederPlugin command")
-        return
+    def command(self, sender, room_id, body, handler = None):
+        self.logger.debug("WKBotsFeederPlugin command: %s" % body)
+        if len(body) == 0:
+            return
+        command = body.split()
+        if command[0].lower() == "mute":
+            self.command_mute(sender, room_id, command[1:], handler)
+
+    def command_mute(self, sender, room_id, command, handler):
+        # Command: mute <builder-name> [ON|OFF].
+        def usage():
+            self.bot.send_html(room_id, "<p>mute <builder-name> [ON|OFF]<p>");
+        if len(command) == 0:
+            usage()
+            return
+        builders = self.settings["builders"]
+        builderName = command[0]
+        if builderName not in builders.keys():
+            usage()
+            return
+        value = (command[1] if command[1:] else "on").lower()
+        if value not in ["on", "off"]:
+            usage()
+            return
+        if value == "on":
+            builders[builderName]["mute"] = True
+        else:
+            builders[builderName].pop("mute")
+        self.bot.send_html(room_id, "<p>Room %s was set to mute %s</p>" % (room_id, value))
 
     def help(self, sender, room_id, handler):
```

Comment:
You could make the [`def usage()`](https://github.com/psaavedra/matrix-bot/pull/56/files#diff-9210bba16129d7d645d5fae49dfab47565db1b8999336573faa6a774a05e6d30R130) a class method and reuse it here.

Or just:
``` python
    def help(self, sender, room_id, handler):
        self.logger.debug("WKBotsFeederPlugin")
        message = "blah ..."
        handler(room_id, message)
```

The rationale behind this is to get this command help listed by the `help` command.